src/main/java/com/lorepo/icplayer/client/utils/DevicesUtils.java

```diff
@@ -4,6 +4,11 @@ public class DevicesUtils {
 	public static native boolean isSafariMobile() /*-{
 		return $wnd.MobileUtils.isSafariMobile($wnd.navigator.userAgent);
 	}-*/;
+
+	public static native boolean isIPod() /*-{
```

Comment:
Metoda nie jest wykorzystywana. Zazwyczaj lepiej nie dodawać, czegoś co nie jest wykorzystywane. Jeżeli wróci to proszę o usunięcie